src/Core/src/Dispatching/DispatcherProvider.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.ComponentModel;
+
+namespace Microsoft.Maui.Dispatching
+{
+	public partial class DispatcherProvider : IDispatcherProvider
+	{
+		// this is mainly settable for unit testing purposes
+		static IDispatcherProvider? s_current;
+
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public static IDispatcherProvider Current =>
```

Comment:
internal?

Reply:
This is a real boy, so I am no longer going tp hide him :)